genvm/cmd/vestingmulti/example.json

```diff
@@ -0,0 +1,16 @@
+{
+    "keys": [
+        "33f8d3b3a9fd558ccef98e9693b6b264e5a593ea096c8984012e94c6e7f72692",
+        "93f801839a7e709f4c270370665ee71b7d86b5f65cb816559503b311d3213962"
+    ], 
+    "required": 1, 
+    "total": 100000e9
+}
```

Comment:
```suggestion
},
```

Reply:
it is not an array, decoder goes through them as separate json objects